src/js/services/conversation.js

```diff
@@ -15,6 +15,59 @@ import { hasLinkableChannels, getLinkableChannels, isChannelLinked } from '../ut
 import { CONNECT_NOTIFICATION_DELAY_IN_SECONDS } from '../constants/notifications';
 import { getUserId } from './user';
 
+const postSendMessage = (dispatch, getState, message) => {
```

Comment:
Then you do `dispatch(postSendMessage(message))` whererever you need to execute.